core/test/unit/services/auth/session/index_spec.js

```diff
@@ -0,0 +1,285 @@
+const sessionService = require('../../../../../server/services/auth/session'),
+    SessionStore = require('../../../../../server/services/auth/session/store'),
+    config = require('../../../../../server/config'),
+    models = require('../../../../../server/models'),
+    {BadRequestError, UnauthorizedError, InternalServerError} = require('../../../../../server/lib/common/errors'),
+    sinon = require('sinon'),
+    should = require('should');
+
+describe('Session Service', function () {
+    let sandbox;
+    before(function () {
+        models.init();
+        sandbox = sinon.sandbox.create();
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+    });
+
+    const fakeReq = function fakeReq() {
+        return {
+            session: {
+                destroy() {}
+            },
+            body: {},
+            get() {}
+        };
+    };
+
+    const fakeRes = function fakeRes() {
+        return {
+            sendStatus() {}
+        };
+    };
+
+    describe('createSession', function () {
+        it('calls next with a UnauthorizedError if there is no body', function (done) {
+            const req = fakeReq();
+            delete req.body;
+            sessionService.createSession(req, fakeRes(), function next(err) {
+                should.equal(err instanceof UnauthorizedError, true);
```

Comment:
That is cool, i always used

`(err instanceof common.errors. UnauthorizedError).should.be.true()`

But yours is much shorter, nice 😁

Reply:
Thanks :dancer: